### PR TITLE
Merged name info to one field and adjusted the mandatory fields

### PR DESCRIPTION
**Pull request by dkoutsonikolis**
…ory fields

## Description

Issue: https://github.com/swiftss-org/registry-api/issues/25

## Checklist

- [x] I have reviewed my code and checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [x] I have updated the documentation (**SwaggerHub**, **README.md**, **Wiki**, etc.) for the changes I have made

### CODE REVIEW

File: tmh_registry/registry/api/serializers.py [hunk 1]
```diff
@@ -42,9 +41,11 @@ class Meta:
     def to_representation(self, instance):
         data = super(ReadPatientSerializer, self).to_representation(instance)
 
-        data["national_id"] = int(data["national_id"])
-        data["phone_1"] = int(data["phone_1"])
-        data["phone_2"] = int(data["phone_2"])
+        data["national_id"] = (
+            int(data["national_id"]) if data["national_id"] else None
+        )
+        data["phone_1"] = int(data["phone_1"]) if data["national_id"] else None
```
Comment (dpapaspyros) [inline tmh_registry/registry/api/serializers.py:47]: @dkoutsonikolis Maybe you meant something like `int(data["phone_1"]) if data["*phone_1*"] else None`? Ideally, I would argue against storing national ID & phones as numbers, but that's another discussion.

Comment (dkoutsonikolis) [reply tmh_registry/registry/api/serializers.py:47]: You are right, this is a mistake due to copy-paste - I will fix it. The national ID and phiones are stored in CharFields.

Comment (dkoutsonikolis) [reply tmh_registry/registry/api/serializers.py]: Fixed!

**Review by StefDoulfis — APPROVED** [review]
Looks good

**Pull request merged**